---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Metagenomics reveals impact of geography and acute diarrheal disease on the
  Central Indian human gut microbiome
subtitle: ''
summary: ''
authors:
- Tanya M. Monaghan
- Tim J. Sloan
- Stephen R. Stockdale
- Adam M. Blanchard
- Richard D. Emes
- Mark Wilcox
- Rima Biswas
- Rupam Nashine
- Sonali Manke
- Jinal Gandhi
- Pratishtha Jain
- Shrejal Bhotmange
- Shrikant Ambalkar
- Ashish Satav
- Lorraine A. Draper
- Colin Hill
- Rajpal Singh Kashyap
tags: []
categories: []
date: '2020-05-01'
lastmod: 2021-01-13T13:38:20Z
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-01-13T13:38:20.587737Z'
publication_types:
- '2'
abstract: 'Background: The Central Indian gut microbiome remains grossly understudied.
  Herein, we sought to investigate the burden of antimicrobial resistance and diarrheal
  diseases, particularly Clostridioides difficile, in rural-agricultural and urban
  populations in Central India, where there is widespread unregulated antibiotic use.
  We utilized shotgun metagenomics to comprehensively characterize the bacterial and
  viral fractions of the gut microbiome and their encoded functions in 105 participants.
  Results: We observed distinct rural-urban differences in bacterial and viral populations,
  with geography exhibiting a greater influence than diarrheal status. Clostridioides
  difficile disease was more commonly observed in urban subjects, and their microbiomes
  were enriched in metabolic pathways relating to the metabolism of industrial compounds
  and genes encoding resistance to 3rd generation cephalosporins and carbapenems.
  By linking phages present in the microbiome to their bacterial hosts through CRISPR
  spacers, phage variation could be directly related to shifts in bacterial populations,
  with the auxiliary metabolic potential of rural-associated phages enriched for carbon
  and amino acid energy metabolism. Conclusions: We report distinct differences in
  antimicrobial resistance gene profiles, enrichment of metabolic pathways and phage
  composition between rural and urban populations, as well as a higher burden of Clostridioides
  difficile disease in the urban population. Our results reveal that geography is
  the key driver of variation in urban and rural Indian microbiomes, with acute diarrheal
  disease, including C. difficile disease exerting a lesser impact. Future studies
  will be required to understand the potential role of dietary, cultural, and genetic
  factors in contributing to microbiome differences between rural and urban populations.'
publication: '*Gut Microbes*'
url_pdf: https://www.tandfonline.com/doi/full/10.1080/19490976.2020.1752605
doi: 10.1080/19490976.2020.1752605
---
